### ISSUE TITLE
Download to *.csv

### ISSUE THREAD

**Issue by rubenbaetens** [opengridcc/opengrid#75]
It would be convenient to have a 'download-to-csv' function to allow easy download of flukso data to an ordinary *.csv file, without needing knowledge of the tmpo or api properties.

**Comment by JrtPec** [comment]
You can put `.to_csv('filename.csv')` behind any data frame, so you could do

```
hp = houseprint.Houseprint()
hp.init_tmpo()
hp.sync_tmpos()
hp.get_data().to_csv('filename.csv')
```

and have all data in one big CSV

We could perhaps have a default `autosync=True` in the Houseprint constructor, than we can omit the `init_tmpo` and `sync_tmpos`. @saroele what do you think?

**Comment by saroele** [comment]
I think the point of this issue is to give the option to opengrid participants (not only coders) to get their data in csv.  I know one participant who says he has no understanding of code, but he would like to have his own data to play with, and that would maybe lead to some ideas for analyses to do. 

And @JrtPec I agree that a default `autosync=True` sounds like a good idea, but that's besides this issue.

**Comment by saroele** [comment]
I think it's  nice exercise (for me or anyone who wants to learn) to make a section on the website where individuals can fill a start end end-date and download a csv with their own data.

**Comment by JrtPec** [comment]
I see. So we'd have a form where they can fill in their own SensorID and Token? This could indeed be a nice Flask exercise :-)

**Comment by saroele** [comment]
yes, or a site-level login.  To be discussed on the next meeting!

**Comment by JrtPec** [comment]
Last week I implemented a simple form on the website to lookup Fluksometers. This works fine, so I think we have all components in place to implement a second form to download data in csv.

**Comment by JrtPec** [comment]
Implemented by opengridcc/website#37
